recipes/fftw/all/conanfile.py

```diff
@@ -70,64 +87,88 @@ def generate(self):
         tc.variables["ENABLE_OPENMP"] = self.options.openmp
         tc.variables["ENABLE_THREADS"] = self.options.threads
         tc.variables["WITH_COMBINED_THREADS"] = self.options.get_safe("combinedthreads", False)
-        tc.variables["ENABLE_FLOAT"] = self.options.precision == "single"
-        tc.variables["ENABLE_LONG_DOUBLE"] = self.options.precision == "longdouble"
         tc.variables["ENABLE_SSE"] = self.options.simd == "sse"
         tc.variables["ENABLE_SSE2"] = self.options.simd == "sse2"
         tc.variables["ENABLE_AVX"] = self.options.simd == "avx"
         tc.variables["ENABLE_AVX2"] = self.options.simd == "avx2"
         tc.generate()
 
+    @property
+    def build_folder(self):
+        bf = super().build_folder
+        return os.path.join(bf, self._current_precision) if self._current_precision else bf
+
+    @property
+    def _all_precisions(self):
+        return [p for p in ALL if self.options.get_safe(f"precision_{p}")]
+
     def build(self):
+        def on_off(value):
+            return "ON" if value else 'OFF'
+
         apply_conandata_patches(self)
-        cmake = CMake(self)
-        cmake.configure()
-        cmake.build()
+        for self._current_precision in self._all_precisions:
+            cmake = CMake(self)
+            variables = {
+                "ENABLE_FLOAT": on_off(self._current_precision == SINGLE),
+                "ENABLE_LONG_DOUBLE": on_off(self._current_precision == LONGDOUBLE),
+                "EENABLE_QUAD_PRECISION": on_off(self._current_precision == QUAD)
```

Comment:
Typo here? CMake tells me that `EENABLE_QUAD_PRECISION` is not used. Shouldn't it be `ENABLE_QUAD_PRECISION`?

Reply:
yes, typo :(